third_party/redis_module/redismodule.h

```diff
@@ -0,0 +1,1271 @@
+#ifndef REDISMODULE_H
```

Comment:
Please add license notice here, refere to https://github.com/redis/redis/blob/unstable/COPYING

Reply:
done